Doc/reference/lexical_analysis.rst

```diff
@@ -758,6 +758,16 @@ include expressions.
    >>> foo.__doc__ is None
    True
 
+F-strings can also help with "printf-style" debugging.
```

Comment:
I think this needs to at least describe the feature. See my 2 points in bpo-41045 for what I think needs to be included.